client/src/api/block_builder/input_selection/new/burn.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::{HashMap, HashSet};
+
+use primitive_types::U256;
+
+use crate::block::output::{AliasId, FoundryId, NftId, TokenId};
+
+/// A type to explicit what needs to be burned during input selection.
+#[derive(Default)]
```

Comment:
Should this have Deserialize derived, so the bindings can provide it as JSON without us requiring an additional type? Serde rename to camelCase should then also be added

Reply:
Sure